pkgs/tools/games/jpsxdec/default.nix

```diff
@@ -0,0 +1,80 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, jdk
+, jre ? jdk
```

Comment:
```suggestion
```

Reply:
This was deliberate, it needs to be built with openjdk8, but it can be run with any. I'd prefer to run it with the default jre because of HiDPI, etc.

Reply:
@SuperSandro2000 thoughts?

Reply:
Please add a comment with that very odd usecase.

Reply:
Done